addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -71,128 +69,100 @@
+                                </button>
+                            </div>
                         </div>
-                    </td>
-                </tr>
-                <tr t-if="!['tags', 'selection'].includes(state.propertyDefinition.type)
+                </div>
+                <div t-if="!['tags', 'selection'].includes(state.propertyDefinition.type)
                     and (!['many2one', 'many2many'].includes(state.propertyDefinition.type)
                     || (state.propertyDefinition.comodel and state.propertyDefinition.comodel.length))"
-                    class="o_field_property_definition_value">
-                    <td class="o_td_label align-top pe-2">
-                        <label t-att-for="getUniqueDomID('default')">
-                            <b t-if="state.propertyDefinition.type === 'boolean'">
-                                Default State
-                            </b>
-                            <b t-else="">
-                                Default Value
-                            </b>
-                        </label>
-                    </td>
-                    <td>
-                        <PropertyValue
-                            id="getUniqueDomID('default')"
-                            readonly="props.readonly"
-                            canChangeDefinition="props.canChangeDefinition"
-                            type="state.propertyDefinition.type"
-                            string="state.propertyDefinition.string"
-                            value="state.propertyDefinition.default"
-                            comodel="state.propertyDefinition.comodel || ''"
-                            domain="state.propertyDefinition.domain || '[]'"
-                            context="props.context"
-                            onChange.bind="onDefaultChange"
-                        />
-                    </td>
-                </tr>
-                <tr t-if="!props.hideKanbanOption" class="o_field_property_definition_kanban">
-                    <td class="o_td_label align-top pe-2">
-                        <label t-att-for="getUniqueDomID('kanban')">
-                            <b>View In Kanban</b>
-                        </label>
-                    </td>
-                    <td>
-                        <CheckBox
-                            id="getUniqueDomID('kanban')"
-                            value="props.propertyDefinition.view_in_kanban"
-                            disabled="props.readonly"
-                            onChange.bind="onViewInKanbanChange"
-                        />
-                    </td>
-                </tr>
-            </table>
+                    class="o_field_property_definition_value d-contents mb-3 mb-sm-0">
+                    <label t-att-for="getUniqueDomID('default')" class="o_form_label align-self-center text-900">
+                        <b t-if="state.propertyDefinition.type === 'boolean'">
```

Comment:
Since the `<b>` tag remain untouched, it's better to keep it outside the condition.
Scenario: let's assume that in the future we want to replace the label using xpath, using your implementation we'll have to replace both `<b>`... this way just one ;) 

```suggestion
                        <b>
                            <t t-if="state.propertyDefinition.type === 'boolean'">Default State</t>
                            <t t-else="">Default Value</t>
                         </b>
                                
```